client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.tsx

```diff
@@ -43,47 +46,81 @@ const PatternLargePreview = ( {
 	isNewSite,
 }: Props ) => {
 	const translate = useTranslate();
-	const hasSelectedPattern = header || sections.length || footer;
+	const hasSelectedPattern = Boolean( header || sections.length || footer );
 	const frameRef = useRef< HTMLDivElement | null >( null );
 	const listRef = useRef< HTMLUListElement | null >( null );
 	const [ viewportHeight, setViewportHeight ] = useState< number | undefined >( 0 );
 	const [ device, setDevice ] = useState< string >( 'computer' );
-	const [ blockGap ] = useGlobalStyle( 'spacing.blockGap' );
+	const [ zoomOutScale, setZoomOutScale ] = useState( 1 );
 	const [ backgroundColor ] = useGlobalStyle( 'color.background' );
-	const [ patternLargePreviewStyle, setPatternLargePreviewStyle ] = useState( {
-		'--pattern-large-preview-background': backgroundColor,
-	} as CSSProperties );
+	const patternLargePreviewStyle = useMemo(
```

Comment:
Super nit: I think `useMemo()` here is unnecessary as `patternLargePreviewStyle` is not props / dependency of anything.

Reply:
It's the style prop of the `ul` element 😆